### DISCUSSION TITLE
Multiple laps adds to ride length stat

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#21764] [Feature requests]
currently the ride length is unaffected by multiple laps, i suggest each laps length be added accumulatively

**Comment by HtotheTML** [comment]
Wouldn't that be an easy exploit for any objective revolving around building coasters of a certain length?

**Reply by 733737** [reply]
how is it an exploit when the coaster is that long?

**Reply by HtotheTML** [reply]
The challenge revolves around building coasters with a certain length of track, sometimes with fairly limited space. You wouldn't need nearly as much space or money to achieve that length if you can just build a fairly short track and have it go through several laps.

**Reply by HtotheTML** [reply]
It would make sense for the ride's runtime to go up with multiple laps, but the physical length of the ride's track shouldn't also go up with multiple laps.

**Comment by ZehMatt** [comment]
The ride length is the length of the tracks and not the distance travelled in total, I don't see any reason to change that.
